slugs.md

```diff
@@ -437,6 +437,7 @@ update the script at 'scripts/release/update-slugs-table.js'.
 | `Cycling '74` | `cycling74` |
 | `Cypress` | `cypress` |
 | `Cytoscape.js` | `cytoscapedotjs` |
+| `D-EDGE` | `dedge` |
```

Comment:
@ericcornelissen, should this be in the PR?

Reply:
It's not needed but it's also not a problem

Reply:
There is a lot of similar case currently:
```
A-Frame 	aframe
Audio-Technica 	audiotechnica
Co-op 	coop
Conda-Forge 	condaforge
````

Reply:
The values are fine, it's just that you don't need to include this in your PR as this file is automatically updated when there's a new release. But including won't interfere with that automation so it's also not a problem if it's included

Reply:
Alright

Reply:
Cool, then i can merge it :)